configure.ac

```diff
@@ -630,13 +622,8 @@ fi
 
 AC_CHECK_FUNCS([mblen], [], [HAVE_MBLEN=1], [])
 
-rm -f Makefile
-if test "$enable_maintainer_mode" = yes ; then
-	AC_MSG_NOTICE(creating maintainer Makefile)
-	ln -s maintainer.mak Makefile
-else
-	AC_CONFIG_FILES(Makefile ctags.1)
-fi
+AC_CONFIG_FILES([Makefile])
+AC_CONFIG_FILES([ctags.1])
```

Comment:
Thank you for pointing it out.
To tell the truth, I separated them unconsciously.

I prefer separated autoconf macro calls, the reason is, to simplify autoconf `--trace` outputs.
If not familiar autoconf `--trace` option, try in ctags git working tree:
```
$ autoconf --trace='AH_TEMPLATE:$1'
```
It outputs cpp macro list `#define`-ed or `#undef`-ed in config.h.

If AC_CONFIG_FILES call is unseparated,
`$ autoconf --trace='AC_CONFIG_FILES:$1'` outputs:
```
Makefile ctags.1
```
but, separeated:
```
Makefile
ctags.1
```
the latter is easy to process, a little.

As spearated AC_CONFIG_FILES calls are not popular,
shoud I revert to unseparated call?